webapp/webpack.config.js

```diff
@@ -0,0 +1,42 @@
+var path = require('path');
```

Comment:
You might want to sync this with https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/webapp/webpack.config.js